themes/src/main/resources/theme/base/account/messages/messages_en.properties

```diff
@@ -337,15 +337,15 @@ sendVerficationCode=Send Verification Code
 enterYourVerficationCode=Enter your verification code
 
 #Authenticator - backup Code setup
-authenticatorBackupCodesSetupTitle=Backup Codes Setup
-backupcodesIntroMessage=If you lose access to your phone, you can still log into your account through backup codes. Keep them somewhere safe and accessible.
+authenticatorBackupCodesSetupTitle=Recovery Authentication Codes Setup
+backupcodesIntroMessage=If you lose access to your phone, you can still log into your account through recovery codes. Keep them somewhere safe and accessible.
```

Comment:
I can't see where quite a few of these message keys are used. Like backupcodesIntroMessage and backupCodesTips?

Reply:
Those where old messages from older implementation, I can remove the messages as part of this PR

Reply:
Either remove them not or not update them. I suspect these where used by OTP a long time ago.